src/celo/types.ts

```diff
@@ -158,17 +181,32 @@ export type RpcTransactionRequestCIP64 = TransactionRequestBase<
   ExactPartial<FeeValuesEIP1559<Quantity>> & {
     accessList?: AccessList | undefined
     feeCurrency?: Address | undefined
+    maxFeeInFeeCurrency?: undefined
     gatewayFee?: undefined
     gatewayFeeRecipient?: undefined
     type?: '0x7b' | undefined
   }
 
+export type RpcTransactionRequestCIP66 = TransactionRequestBase<
+  Quantity,
+  Index
+> &
+  ExactPartial<FeeValuesEIP1559<Quantity>> & {
+    accessList?: AccessList | undefined
+    feeCurrency?: Address | undefined
+    maxFeeInFeeCurrency?: Quantity | undefined
+    gatewayFee?: undefined
```

Comment:
Looks like `gatewayFee` and `gatewayFeeRecipient` are still optional arguments here. 

Probably not in scope to remove them all in this PR, but might make sense to not add `gatewayFee` and `gatewayFeeRecipient` to CIP66 related types.

What do you think?

Reply:
they're getting removed as we're removing CIP-42, need to rebase the branch